scale-tests/deploy_dsengine.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -x
+
+create_service_account() {
+  SERVICE_ACCOUNT="${1}"
+  SECRET_NAME="${SERVICE_ACCOUNT}-secret"
+
+  dcos security org service-accounts delete "${SERVICE_ACCOUNT}"
+  dcos security secrets delete "${SECRET_NAME}"
+
+  dcos security org service-accounts keypair private.pem public.pem
+  dcos security org service-accounts create \
+       -p public.pem \
+       -d "Service account for ${SERVICE_ACCOUNT}" "${SERVICE_ACCOUNT}"
+  dcos security secrets create-sa-secret \
+       --strict private.pem \
+       "${SERVICE_ACCOUNT}" \
+       "${SECRET_NAME}"
+
+  rm -f private.pem public.pem
+}
+
+grant_permissions() {
+  SERVICE_ACCOUNT="${1}"
+
+  echo "Granting permissions to Service Account ${SERVICE_ACCOUNT}"
+
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:master:task:user:nobody" \
+       create
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:agent:task:user:nobody" \
+       create
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:secrets:list:default:__dcos_base64__hdfs_jupyter_keytab" \
+       read
+}
+
+grant_spark_permissions() {
+  SERVICE_ACCOUNT=$1
+  echo "Granting Spark permissions to Jupyter Service Account ${SERVICE_ACCOUNT}"
+
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:master:framework:role:${SERVICE_ACCOUNT}" \
+       create
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:master:reservation:role:${SERVICE_ACCOUNT}" \
+       create
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:master:reservation:principal:${SERVICE_ACCOUNT}" \
+       delete
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:master:volume:role:${SERVICE_ACCOUNT}" \
+       create
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:master:volume:principal:${SERVICE_ACCOUNT}" \
+       delete
+
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:master:task:role:${SERVICE_ACCOUNT}" \
+       create
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:master:task:principal:${SERVICE_ACCOUNT}" \
+       create
+  dcos security org users grant "${SERVICE_ACCOUNT}" \
+       "dcos:mesos:master:task:app_id:data-services/jupyter" \
+       create
+}
+create_service_account data_services__jupyter
+
+grant_permissions data_services__jupyter
+grant_spark_permissions data_services__jupyter
+
+dcos package install --yes data-science-engine \
+     --options=scale-tests/configs/jupiter-options.json
```

Comment:
can we rename the file to `jupyter-options.json`?

Reply:
Yeah. Even better, `dsengine-options.json`.

Reply:
https://github.com/mesosphere/spark-build/pull/550/commits/38bbbd27fa7bb34b43e4b094ff7634cd9d4f7754